src/charactersheet/components/named-fields-well/index.js

```diff
@@ -0,0 +1,36 @@
+import { components, observable } from 'knockout';
+
+import template from './index.html';
+
+class NamedFieldsWell {
```

Comment:
Could you add some docs explaining the fields needed for the component and what their purpose is?
This would help us in the future when we don't remember things related to this component.